sitegen-maven-plugin/src/main/java/io/helidon/build/sitegen/maven/AbstractAsciiDocMojo.java

```diff
@@ -264,8 +265,12 @@ void processFile(
      * @return PathMatchers for the globs
      */
     private static Stream<PathMatcher> pathMatchers(Path inputDirectory, String[] globs) {
-          return Arrays.stream(globs)
+        return Arrays.stream(globs)
                 .map(glob -> {
+                    if (WINDOWS) {
+                        String pattern = "glob:" + inputDirectory + File.separator + glob.replace("/", "\\");
+                        return FileSystems.getDefault().getPathMatcher(pattern.replace("\\", "\\\\"));
```

Comment:
`Matcher.quoteReplacement(String.valueOf(inputDirectory));` may also work

Reply:
Sure. Thanks for pointing on it. I am not sure that using it brings any advantages over `String.replace`.